acpi.adoc

```diff
@@ -64,9 +69,16 @@ objects.
 | [[acpi-tad]] AML_070 | Systems with a Real-Time Clock on an OS-managed bus (e.g. I2C, subject to arbitration issues due to access to the bus by the OS) MUST implement the Time and Alarm Device (TAD).
 2+| _Also see <<uefi-rtc, URT_020>>_.
 | AML_080 | Systems implementing a TAD must be functional without additional system-specific OS drivers.
+| [[acpi-irq-gsb]] AML_090 | PLIC and APLIC devices must support
+**_GSB** method. <<acpi-ids, See RVI ACPI IDs>>.
```

Comment:
Suggest this as being _GSB (Global System Interrupt Base), with no need for bolding as we haven't done that elsewhere

Reply:
Even better would be something like Global System Interrupt Base (_GSB, cite:[ACPI] Section 6.2.7).

Reply:
Sure

Reply:
So if we move away from MADT entries, then these devices would also return _MAT, right?

Reply:
_MAT is not really mandatory. Even IOAPIC namespace device don't need _MAT. OS will look at the MADT entry directly using _GSB.